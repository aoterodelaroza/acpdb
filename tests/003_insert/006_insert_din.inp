## check: 006_insert_din.out -a1e-10
## delete: 006_insert_din.db
## labels: regression quick

system rm -f 006_insert_din.db
connect 006_insert_din.db

INSERT METHOD test
END

insert litref bibtex ../zz_source/bsip2.bib
insert set s22
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method test
end
