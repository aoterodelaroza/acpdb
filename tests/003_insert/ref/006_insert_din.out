* SYSTEM: rm -f 006_insert_din.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 006_insert_din.db
Creating skeleton database 

* INSERT: insert data into the database (METHOD)

* INSERT: insert data into the database (LITREF)

* INSERT: insert data into the database (SET)

